### DISCUSSION TITLE
Boundary Coordinates

### DISCUSSION THREAD

**Discussion by bkfichter** [domlysz/BlenderGIS#454] [General]
Is it possible to add the final boundary coordinates to the custom data? Is there a way to get that info already?

**Comment by domlysz** [comment]
for what operator?

**Reply by bkfichter** [reply]
So immediately after selecting a region and hitting "E", it would be great to have the N,E,S,W bounding coordinates of the selected area in the data somewhere. It can be used for rough location plotting using liner interpolation in a game engine. 
To be clear, I added an image below showing the boundary and coordinates. (doesn't have to be in the viewport, but in the data.)

This is some amazing work you've done.

**Reply by bkfichter** [reply]
![Capture](https://user-images.githubusercontent.com/20936686/111353736-93805200-8653-11eb-95c7-4374bb1cef7c.PNG)

**Comment by domlysz** [comment]
there is a a tool in geoscene panel that print the geo-coordinate of the mouse pointer
![image](https://user-images.githubusercontent.com/5639024/111587859-0e1baf80-87c3-11eb-9cc9-1fb3b315ad37.png)

or you can compute it yourself taking the coord of the scene center (also specified in the geoscene panel) and apply this delta to the coords of each corners (select the vertex in edit mode to get it's coord).

**Reply by bkfichter** [reply]
If you mean toggling the Geo-coordinates tool to get the mouse location coordinates, it seems to be off unless I am using it incorrectly. The center is correct, but the values seem to be wrong as you move away from the center. Am I using it correctly?
![delete](https://user-images.githubusercontent.com/20936686/111720067-317a4500-882b-11eb-8162-80eb1a0e7e07.png)

**Reply by bkfichter** [reply]
@domlysz I put this aside for a bit because I couldn't figure out what to do based on your suggestion. Back to it though. Still trying to figure out how to get the boundary coordinates. Can you please provide more detail? (C4D have DemEarth which is pretty good and provides all the coordinate data, but I'd like to stay in Blender.)

**Comment by bkfichter** [comment]
Years later and I still don't have an answer for this. Trying to use this tool again, Doesn't look like there have been updates to this. Anyone else out there know how to do this?

**Comment by rumicuna** [comment]
I'm also trying to use the resulting models in a game. I'd like to convert gps coordinates to model coordinates in order to place objects at the right location, and in order to do this I need to corner coordinates so I can interpolate.
